main.go

```diff
@@ -152,8 +153,9 @@ func main() {
 		return err == nil
 	}
 	servicesToWait["ta2"] = func() bool {
-		err := solutionClient.Hello()
-		return err == nil
+		versionNumber := solutionClient.Hello()
+		ta2Version = versionNumber
+		return versionNumber.(string)
```

Comment:
I think I understand what you are trying to do here. You still want to have the return statement be `return err == nil` since the function needs to return a bool indicating whether it succeeded or not.

Reply:
I am trying to test if the version number is a String. But now that I'm writing this I guess in a typed language this is an irrelevant test...

I want to test if we got an answer from the handshake.